.gitignore

```diff
@@ -0,0 +1,169 @@
+# Byte-compiled / optimized / DLL files
```

Comment:
Did you intentionally add `.gitignore`? If yes, I'd suggest cleaning it up from unnecessary entries before merging.

Reply:
Ack. This was from the boilerplate .gitignore from `pdm init` with some extras added from poetry and pixi. Some of it's best practices/common stuff and seemed harmless but I'll delete the unnecessary stuff.